src/main/webapp/WEB-INF/tags/wrapper.tag

```diff
@@ -80,6 +91,72 @@
+         aria-label="Accept all cookies" id="accept-all-cookies-success">
+        <div class="govuk-cookie-banner__message govuk-width-container">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <div class="govuk-cookie-banner__content govuk-p-size-override">
+                        <p class="govuk-body govuk-p-size-override"><spring:message code="public.cookie.banner.accepted.text_0001"/></p>
+                        <p class="govuk-body govuk-p-size-override"><spring:message code="public.cookie.banner.accepted.text_0002"/> <a href="/cookie-preferences"><spring:message code="public.cookie.banner.accepted.text_0003"/></a>
+                            <spring:message code="public.cookie.banner.accepted.text_0004"/>
+                        </p>
+                        <p id="cookie-accept-all-success-banner-hide" class="govuk-body govuk-button-group">
+                            <a class="govuk-button govuk-p-size-override"><spring:message code="public.cookie.banner.accepted.text_0005"/></a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div id="cm_cookie_notification" class="govuk-visually-hidden">
+        <div class="govuk-cookie-banner " data-nosnippet role="region" aria-label="Cookies on hmcts-access.service.gov.uk">
+            <div class="govuk-cookie-banner__message govuk-width-container">
+                <div class="govuk-grid-row">
+                    <div class="govuk-grid-column-two-thirds">
+                        <h2 class="govuk-cookie-banner__heading govuk-heading-m govuk-h-size-override"><spring:message code="public.cookie.banner.text_0001"/></h2>
+                        <div class="govuk-cookie-banner__content govuk-p-size-override">
+                            <p><spring:message code="public.cookie.banner.text_0002"/></p>
+                            <p><spring:message code="public.cookie.banner.text_0003"/></p>
+                        </div>
+                    </div>
+                </div>
+                <div class="govuk-button-group govuk-p-size-override">
+                    <button value="accept" type="button" name="cookies" class="govuk-button govuk-p-size-override" data-module="govuk-button" id="cookie-accept-submit">
+                        <spring:message code="public.cookie.banner.text_0004"/>
+                    </button>
+                    <button value="reject" type="button" name="cookies" class="govuk-button govuk-p-size-override" data-module="govuk-button" id="cookie-reject-submit">
+                        <spring:message code="public.cookie.banner.text_0005"/>
+                    </button>
+                    <a class="govuk-link govuk-p-size-override" href="${pageContext.request.contextPath}/cookies"><spring:message code="public.cookie.banner.text_0006"/></a>
```

Comment:
Should this link be to `/cookies` or `/cookie-preferences`?

Reply:
This ones supposed to go to cookies for some reason 🤷